modules/optimization/src/optimizationreporters/ParameterMeshOptimization.C

```diff
@@ -256,3 +263,41 @@ ParameterMeshOptimization::parseData(const std::vector<unsigned int> & exodus_ti
 
   return parsedData;
 }
+
+void
+ParameterMeshOptimization::execute()
```

Comment:
IN another PR, I think we need a invOptReporterBase for these repeated implementations like execute and computeObjective that are not needed for other types of optimization.